### PR TITLE
gh: fix dependencies

### PR DESCRIPTION
**Pull request by Valium007**
Package depends on openssh for sshing into codespaces

### CODE REVIEW

**Comment by landfillbaby** [comment]
maybe it should be a recommends or suggests instead

**Comment by Valium007** [comment]
> maybe it should be a recommends or suggests instead

done!

File: packages/gh/build.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ TERMUX_PKG_MAINTAINER="Krishna kanhaiya @kcubeterm"
 TERMUX_PKG_VERSION="2.43.1"
 TERMUX_PKG_SRCURL=https://github.com/cli/cli/archive/v${TERMUX_PKG_VERSION}.tar.gz
```
Comment (licy183) [inline packages/gh/build.sh:6]: Please revbump this package.
`TERMUX_PKG_REVISION=1`

**Comment by licy183** [comment]
Thanks!

**Comment by TomJo2000** [comment]
While we're at it.
Shouldn't `openssh` also be a `TERMUX_PKG_RECOMMENDS` for `git` itself?
Since you can push/pull and sign commits/tags/pushes with it.

**Pull request merged**